pages/authLogout/authLogout.js

```diff
@@ -7,7 +7,7 @@ router.get('/', function(req, res, _next) {
     if (res.locals.authn_provider_name == 'Shibboleth') {
         res.redirect('/Shibboleth.sso/Logout');
     } else {
-        res.redirect('/');
+        res.redirect('/pl/login');
```

Comment:
Does this interfere with homepage cookie-clearing? https://github.com/PrairieLearn/PrairieLearn/blob/517be1007699db7ab2d5c823a5912fa00b39d045/server.js#L228

Reply:
I think this is required so a logout specifically triggers another login. If it didn't and redirected to /, the "if no cookie and devMode, assume dev login" would stop any other login.

I think it's independent from the homepage cookie-clearing, but that's something we'd need to test. I'm not sure how.

Reply:
I'm fine to redirect to `/pl/login`, but that's going to prevent the cookie-clearing code from working. I think we should add some other way of triggering that.

Reply:
I think we might be triggering this because the test that's failing is when it's trying to emulate a student. I'm investigating.

Could logout just clear all of the cookies?

Reply:
Sure, clearing cookies on logout would be fine, but I think you might need to bounce via another page to do it?